pyproject.toml

```diff
@@ -31,6 +31,7 @@ classifiers = [
 ]
 dependencies = [
     "Pillow>=9.3",
+    "av>=10.0.0",
```

Comment:
```suggestion
dependencies = [  # optional: av>=10.0.0
    "Pillow>=9.3",
```

Reply:
suggest making this an optional dependency, so the package still works for image data without it installed

Reply:
So something like this:
```toml
dependencies = [
    "Pillow>=9.3",
    "numpy>=1.20.0",
    "pandas>=1.1.5",
    "tabulate>=0.8.3", # pandas optional dependency for .to_markdown()
    "imagehash>=4.2.0",
    "tqdm>=4.53.0",
    "matplotlib>=3.4",
    "fsspec>=2023.1.0", # latest compatible with Python 3.7
]

[project.optional-dependencies]
huggingface = ["datasets>=2.7.0"]
pytorch = ["torchvision>=0.12.0"]
azure = ["adlfs>=2022.2.0"] # latest compatible with Python 3.7
gcs = ["gcsfs>=2022.1.0"] # latest compatible with Python 3.7
s3 = ["s3fs>=2023.1.0"] # latest compatible with Python 3.7
av = ["av>=10.0.0"]

all = ["cleanvision[huggingface,pytorch,azure,gcs,s3,av]"]
```

Or just don't touch the `pyproject.toml` file at all?

Reply:
This looks good, instead of `av`, you could use the `video` keyword.